README.md

```diff
@@ -242,43 +242,44 @@ PcapPlusPlus currently supports parsing, editing and creation of packets of the
 22. NDP
 23. Raw IP (IPv4 & IPv6)
 24. VRRP (IPv4 & IPv6)
+25. WireGuard
 
 ### Transport Layer (L4)
 
-25. COTP
-26. GTP (v1)
-27. IPSec AH & ESP - parsing only (no editing capabilities)
-28. TCP
-29. TPKT
-30. UDP
+26. COTP
```

Comment:
The numbering here seems to be very unfriendly for new layers. How about changing them all into the non-number list? @seladb

Reply:
I think that it's nice to see the overall number of protocols, but otherwise no real reason...